File: src/main/java/com/example/Nadeuri/likes/dto/response/LikeDeleteResponse.java
package com.example.Nadeuri.likes.dto.response;

import com.example.Nadeuri.likes.LikeEntity;
import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class LikeDeleteResponse {
    private final Long likeId;

    public static LikeDeleteResponse from(final LikeEntity like) {
        return new LikeDeleteResponse(like.getId());
    }
}

File: src/main/java/com/example/Nadeuri/likes/controller/dto/response/LikeDeleteResponse.kt
package com.example.Nadeuri.likes.controller.dto.response

import com.example.Nadeuri.likes.entity.LikeEntity

data class LikeDeleteResponse(
    val likeId: Long,
) {
    companion object {
        fun from(like: LikeEntity): LikeDeleteResponse =
            LikeDeleteResponse(
                likeId = like.id
            )
    }
}